docs/features/kubernetes/authentication.md

```diff
@@ -0,0 +1,35 @@
+---
+id: kubernetes-auth
+title: Kubernetes Authentication
+description: Authentication in kubernetes plugin
```

Comment:
Not sure what this field gets used for, but capitalizing Kubernetes seems like the norm in these docs
```suggestion
description: Authentication in Kubernetes plugin
```